packages/apollo-link-http/README.md

```diff
@@ -69,7 +69,48 @@ client.query({
 })
 ```
 
-## Upgrading from `apollo-fetch` / `apollo-client` 
+## Errors
+The Http Link draws a distinction between client, server and GraphQL errors.
+Server errors can occur in three different scenerios: parse, network and data errors.
+This list describes the different errors and what the response Observable calls:
+
+* Client parse errors: the request body is not-serializable due to circular references for example
+* Server parse errors: the response from the server cannot be parsed((response.json())[https://developer.mozilla.org/en-US/docs/Web/API/Body/json])

```

Comment:
I think this link is mis-formatted - probably worthwhile to look at the deploy preview or run locally